src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      class="big-search"
+      data-qa="big"
+    >
+      <label class="big-search__img" for="search-big"></label>
+      <input
+        type="search"
+        name="search-big"
+        id="search-big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="united-search-input big-search__input"
+        data-qa="keypress"
+      >
```

Comment:
Also, it's possible to wrap input inside the label
```suggestion
      <label class="big-search__img">
        <input
          type="search"
          name="search-big"
          id="search-big"
          placeholder="Try &quot;Los Angeles&quot;"
          class="united-search-input big-search__input"
          data-qa="keypress"
        >
      </label>
```